### DISCUSSION TITLE
Final Version Famidash

### DISCUSSION THREAD

**Discussion by TheBoxGame** [tfdsoft/famidash#248] [General]
Hello, I have a question when can we expect the final version of the game because I liked the game

**Comment by kandowontu** [comment]
1.0 can be released anywhere from this month up to as far as potentially september/october. 1.1 will likely be ready by January/Feb. All rough goals though!

**Reply by TheBoxGame** [reply]
thanks

**Comment by kandowontu** [comment]
woah look at that it released right on schedule!

**Reply by TheBoxGame** [reply]
thanks
